test/modules/DirectMethodSender/DirectMethodResultType.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DirectMethodSender
+{
+    public enum DirectMethodResultType
+    {
+        LegacyDirectMethodTestResult,
```

Comment:
do we need 3 different types for E2E, long haul and connectivity test?

Reply:
Nope. E2E can share the same `LegacyDirectMethodTestResult` with Long haul. 
Since E2E does not check the result string, it only check if there is an event.
The `DirectMethodTestResult` is used for TRC.

Reply:
does it mean even response status code is not 200, e2e test will still pass?

Reply:
That is very suspicious.